framework/interface/templates/home.html

```diff
@@ -3,21 +3,7 @@
 {% block title %}Home Page{% end %}
 
 {% block content %}
-<div class="container-fluid jumbotron">
```

Comment:
Can the info box be there on the left and pie on the right?
Or will it look better if the pie is below the jumbotron? Inputs - @saganshul ?